app/components/CrashLocationBanner.tsx

```diff
@@ -0,0 +1,21 @@
+import Alert from "react-bootstrap/Alert";
+
+interface CrashLocationBannerProps {
+  privateDriveFlag: boolean | null;
+}
```

Comment:
Initially was passing the entire Crash object and thus had this as 
```typescript
interface CrashHeaderProps {
  crash: Crash;
}
```

but then switched to just the private drive flag. I considered then just getting rid of this entire interface and having the types in the function definition, but this felt easier to read. Lemme know if yall prefer the other way. This also feels easier to extend in the future, but I dont anticipate us passing more props into this component.

Reply:
I am a huge fan of passing in what you need and not everything else. I endorse this for clarity and efficiency -- this feels like the way to me.

Reply:
thanks for kicking off this convo. i agree with your approach and started a Slack discussion about some other considerations for how we want to manage our types, prop types, and comments: https://austininnovation.slack.com/archives/CHZE6BC6L/p1733420020334539